example/operator/20-garden.yaml

```diff
@@ -196,6 +196,12 @@ spec:
     # gardenerScheduler:
     #   featureGates:
     #     SomeGardenerFeature: true
+    #   schedulerControllerConfiguration:
+    #     backupBucket:
+    #       concurrentSyncs: 5
+    #     shoot:
+    #       concurrentSyncs: 5
```

Comment:
Didn't we discuss this internally and decided not to expose these fields for now?

Reply:
oh, I missed that 🥲